src/tests/test_gsp.py

```diff
@@ -20,18 +20,18 @@
 
 @freeze_time("2022-01-01")
 def test_read_latest_one_gsp(db_session, api_client):
```

Comment:
hi @peterdudfield, this PR isn't quite ready for review, but I wanted to ask a question about two tests. This one `test_read_latest_one_gsp` throws an error that the index is out of range. 

do I need this test now that the `/gsp/{gsp}/forecast` route has been updated? 

I'll leave my other question on the other problematic test.

Reply:
interesteing, it shouldnt get an error. Whats is the error message?

Reply:
I think ive fixed it

Reply:
this makes sense now. the error message was that there was no object to concatenate.